### PR TITLE
[docs] Update README.md

### PR DESCRIPTION
**Pull request by smousy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
 
 To learn more about the platform, check out our [documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
 
-## Installation
+## Installation and usage
 
-- Quick start using Docker Compose
```
Comment (MksmOrlov) [inline README.md:17]: Why do you remove docker installation. It is just another way to install project. Revert it please

File: README.md [hunk 2]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
 
 To learn more about the platform, check out our [documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
 
-## Installation
+## Installation and usage
```
Comment (MksmOrlov) [inline README.md:15]: There is no information about **usage** of ostis-web-platform. Remove it please

File: README.md [hunk 3]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
...
-    ```
-
-  </details>
-
-  Requirements: you will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+- Natively
 
   ```sh
-  git clone https://github.com/ostis-ai/ostis-web-platform
+  git clone (ostis-web-platform link)
```
Comment (MksmOrlov) [inline README.md:20]:
```suggestion
  git clone https://github.com/ostis-ai/ostis-web-platform
```

File: README.md [hunk 4]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
...
   ./build_kb.sh
-  # Launch knowledge processing machine
-  cd ostis-web-platform/scripts/
   ./run_sc_server.sh
-  # *in another terminal*
-  # Launch semantic web interface at localhost:8000
-  cd ostis-web-platform/scripts/
-  ./run_scweb.sh
+  # in another terminal (ctrl+shift+T)
+  ./run_sc_web.sh
```
Comment (MksmOrlov) [inline README.md:31]:
```suggestion
  # in another terminal (ctrl+shift+T)
  cd  kb/sc_web/scrtipts
  ./run_scweb.sh
```

File: README.md [hunk 5]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
...
+  # open localhost:8000
   ```
-
+
```
Comment (MksmOrlov) [inline README.md:34]: Remove please

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
...
-  # start platform services and run web interface at localhost:8000
-  docker compose up
-  ```
-
-- Native installation
-
-  ```sh
-  # Build knowledge base
-  cd ostis-web-platform/scripts/
+  cd ../sc-machine/scripts
```
Comment (NikitaZotov) [inline README.md:25]: I think we should leave this docs about installation in docker and natively and write new docs about way of installation throught sc-component-manager. Totally, there must be three sections about installation

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -76,6 +76,18 @@ To learn more about the platform, check out our [documentation](https://github.c
   ./prepare.sh
   ```
 
+- Using component manager
```
Comment (MksmOrlov) [inline README.md:79]: There is nothing about using, just installation

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (smousy) [reply README.md:79]: i mean installation method using component manager

**Review by MksmOrlov — APPROVED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**